docs/advanced.md

```diff
@@ -533,10 +533,10 @@ irb(main):004:0> manual_password = 'example-password'
 => "example-password"
 ```
 
-Now call `GitHelper.clone`, which will clone and decrypt the repo for you. Assign the return value to `workspace`, which we'll need later when we re-encrypt:
+Now call `GitHelper.clone`, which will clone and decrypt the repo for you. Assign the return value to `workspace`, which we'll need later when we re-encrypt. As match commonly creates a branch per developer team (the name of the branch being the team identifier), you can optionally pass in the branch as a parameter to the `clone` method (or omit the parameter to use `master`):
 
 ```ruby
-irb(main):005:0> workspace = Match::GitHelper.clone(git_url, shallow_clone, manual_password: manual_password)
+irb(main):005:0> workspace = Match::GitHelper.clone(git_url, shallow_clone, manual_password: manual_password, branch: 'ABCDE12345')
```

Comment:
Can we keep an example without passing `branch: 'ABCDE12345'` in here too? It's good to have examples of both (especially if people aren't super comfortable with Ruby) 🙃

Reply:
Was wary of keeping both the examples so as not to bloat this section with duplication. But let me have a try :)